legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/core/FunctionExecutionSupport.java

```diff
@@ -238,10 +274,8 @@ static Iterable<? extends LegendExecutionResult> executeFunction(FunctionExecuti
         return results;
     }
 
-    static Iterable<? extends LegendExecutionResult> executeQuery(FunctionExecutionSupport executionSupport, SectionState section, String entityPath, Map<String, String> executableArgs, Map<String, Object> inputParameters)
+    static Iterable<? extends LegendExecutionResult> executeQuery(AbstractLSPGrammarExtension extension, SectionState section, String entityPath, Map<String, String> executableArgs, Map<String, Object> inputParameters)
```

Comment:
Lets rollback this change, and all other similar changes.  These utilities should be for function executoin, and not all extensions support this.

Reply:
Done